Sources/ContainerClient/Core/ClientImage.swift

```diff
@@ -289,10 +289,30 @@ extension ClientImage {
         let request = newRequest(.imagePrune)
         let response = try await client.send(request)
         let digests = try response.digests()
-        let size = response.uint64(key: .size)
+        let size = response.uint64(key: .imageSize)
         return (digests, size)
     }
 
+    /// Calculate disk usage for images
+    /// - Parameter activeReferences: Set of image references currently in use by containers
+    /// - Returns: Tuple of (total count, active count, total size, reclaimable size)
+    public static func calculateDiskUsage(activeReferences: Set<String>) async throws -> (Int, Int, UInt64, UInt64) {
```

Comment:
We might wanna give the tuple fields names (or return a struct, either or). `(totalCount: Int, activeCount: Int, totalSize: UInt64, reclaimableSize: UInt64)`